FastAnimations/i18n/zh.json

```
@@ -71,9 +71,8 @@
     "config.fade.name": "淡入淡出",
     "config.fade.tooltip": "屏幕淡入淡出黑色的速度。默认{{defaultValue}}x。",
 
-    // TODO
-    "config.fishing-text.name": "Fishing minigame text",
-    "config.fishing-text.tooltip": "How fast fishing minigame text animations like 'HIT!' and 'PERFECT' play. Default {{defaultValue}}x.",
+    "config.fishing-text.name": "钓鱼小游戏",
+    "config.fishing-text.tooltip": "钓鱼小游戏内部分动画的播放速度，例如“上钩了！”和“完美”。默认{{defaultValue}}x。",
```

```suggestion
    "config.fishing-text.name": "钓鱼小游戏内的文字",
    "config.fishing-text.tooltip": "钓鱼小游戏内部分文字动画的播放速度，例如“上钩了！”和“完美”。默认{{defaultValue}}x。",
```
not clear enough yet